cosign.pub

```diff
@@ -0,0 +1,4 @@
+-----BEGIN PUBLIC KEY-----
```

Comment:
This needs to be placed in the goreleaser artifact from the `COSIGN_PUBLIC_KEY` environment variable.